app/scripts/models/auth_brokers/fx-firstrun-v1.js

```diff
@@ -26,6 +27,21 @@ define(function (require, exports, module) {
       VERIFICATION_COMPLETE: 'verification_complete'
     },
 
+    constructor () {
+      if (this._isInAnIframe()) {
```

Comment:
not sure if this function is available here, you might need to use the options passed into the `auth_broker` instead to determine this

Reply:
@vladikoff , thanks for suggestions working on them now